scipy/optimize/_minimize_constrained.py

```diff
@@ -0,0 +1,485 @@
+        is the algorithm trust-radius. Default is 1e-8.
+    gtol : float, optional
+        Tolerance for termination by the norm of the lagrangian gradient.
+        The algorithm will terminate when both the infinite norm of the
+        lagrangian gradient and the constraint violation are smaller than
+        ``gtol``. Default is 1e-8.
+    sparse_jacobian : {boolean, None}
+        The algorithm uses a sparse representation of the Jacobian if True
+        and a dense representation if False. When ``None`` the algorithm
+        uses the more convenient option (which is not always the more effective
+        one).
+    options : dict, optional
+        A dictionary of solver options. Available options include:
+
+            initial_trust_radius: float
+                Initial trust-region radius. Default is 1.
+            initial_penalty : float
+                Initial penalty for merit function. Default is 1.
+            initial_barrier_parameter: float
+                Initial barrier parameter. Exclusive for ``tr_interior_point``
+                method. By defaut uses 0.1.
+            initial_tolerance: float
+                Initial subproblem tolerance. Exclusive for
+                ``tr_interior_point`` method. By defaut uses 0.1.
+            return_all : bool, optional
+                When ``true`` return the list of all vectors
+                through the iterations.
+            factorization_method : string, optional
+                Method used for factorizing the jacobian matrix.
+                Should be one of:
+
+                - 'NormalEquation': The operators
+                   will be computed using the
+                   so-called normal equation approach
+                   explained in [1]_. In order to do
+                   so the Cholesky factorization of
+                   ``(A A.T)`` is computed. Exclusive
+                   for sparse matrices.
+                - 'AugmentedSystem': The operators
+                   will be computed using the
+                   so-called augmented system approach
+                   explained in [1]_. Exclusive
+                   for sparse matrices.
+                - 'QRFactorization': Compute projections
+                   using QR factorization. Exclusive for
+                   dense matrices.
+                - 'SVDFactorization': Compute projections
+                   using SVD factorization. Exclusive for
+                   dense matrices.
+
+                The factorization methods 'NormalEquation' and
+                'AugmentedSystem' should be used when
+                ``sparse_jacobian=True`` and, on the other
+                hand, the methods 'QRFactorization' and
+                'SVDFactorization' when ``sparse_jacobian=False``.
+
+    callback : callable, optional
+        Called after each iteration:
+
+            callback(OptimizeResult state) -> bool
+
+        If callback returns true the algorithm execution is terminated.
```

Comment:
``True`` here and probably elsewhere. Whatever you choose, try to be consistent.